.github/workflows/windows-build.yml

```diff
@@ -23,5 +23,11 @@ jobs:
         run: ./gradlew test
       - name: Integration tests
         run: ./gradlew integrationTest
+      - name: Upload test reports
```

Comment:
There's really no need to upload test results, as you can view them with the help of a build scan. The build scan link is available at the end of each workflow run, on success and on failure. I'd say remove this change.